zoom.sh

```diff
@@ -9,4 +9,10 @@
 # https://github.com/flathub/us.zoom.Zoom/issues/445
 rm -f $HOME/.zoom/data/cefcache/*/SingletonSocket
 
+# Support for Freedesktop.org user directory layout
+# Verify ~/Documents does not exist, to then create a symbolic link to the xdg documents directory
+if [[ ! -e ~/Documents ]]; then
+	ln -s $(xdg-user-dir DOCUMENTS) ~/Documents
```

Comment:
I'm wondering about quoting and whether it's worth preventing the empty string being produced by accident to not trip up `ln`, e.g.:
```
XDG_DOCUMENTS=$(xdg-documents DOCUMENTS)
XDG_DOCUMENTS=${XDG_DOCUMENTS:=~/Documents}
...
ln -s   "${XDG_DOCUMENTS}"   "~/Documents"
```

Or am I overly defensive about this potential problem?

Reply:
The user-dirs.dirs file only supports path with `$HOME/` as prefix for relative paths and `/` for absolute ones. `xdg-user-dir` falls back to `$HOME`. Flatpak generates a valid file containing all mounted user directories, and in this case it contains xdg-documents. So, `$HOME` would need to be unset and a malformed user-dirs.dirs created for this to be able to be the empty string? Is this something you want to protect against?

Reply:
My thinking was that a future `xdg-user-dir` might misbehave for reasons we cannot necessarily envision yet. It could not return anything or multiple lines, so I thought it could be interesting to defend against some potential problems. But anyway, that's not something to block on.
I'm fine with the proposed change, although I think it's ultimately a problem in the software itself, i.e. Zoom should really be honouring xdg-user-dir rather than us working around it.
Let's let it cook for a couple of days for somebody to speak up. If nobody complains, let's merge it. Feel free to nag.